demo/site/src/news/NewsList.tsx

```diff
@@ -1,22 +1,59 @@
 "use client";
 import { DamImageBlock } from "@src/common/blocks/DamImageBlock";
+import { fetcher } from "@src/util/Fetcher";
 import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
+import useSWR from "swr";
 
-import { GQLNewsListFragment } from "./NewsList.fragment.generated";
+import { GQLNewsIndexPageQuery } from "./NewsList.loader.generated";
+
+export function NewsList({ fallbackData }: { fallbackData: GQLNewsIndexPageQuery["newsList"]["nodes"] }) {
+    const [category, setCategory] = useState<string | undefined>(undefined);
```

Comment:
In most of our applications, this would be a param in the URL. Should we implement it like this in Demo as well.